lib/src/database/db.dart

```diff
@@ -209,7 +210,7 @@ class Db {
       }
 
       if (connection == null) {
-        connection = _masterConnection;
+        connection = _masterConnectionVerified;
```

Comment:
@Pacane, sorry, I didn't understand you. Final resolution - leave everything like it is now?